src/sql/azdata.d.ts

```diff
@@ -2524,42 +2524,38 @@ declare module 'azdata' {
 		 * Azure Key Vault
 		 */
 		AzureKeyVault = 3,
-		/**
-		 * Azure AD Graph
-		 */
-		Graph = 4,
 		/**
 		 * Microsoft Resource Management
 		 */
-		MicrosoftResourceManagement = 5,
+		MicrosoftResourceManagement = 4,
```

Comment:
Don't change these values, that could result in breaking extensions if they're doing some weird stuff with passing in the number values directly. And while that's an edge case, the exact numeric values of the enums shouldn't matter so it's generally best practice to not change these.

You can delete the old value, just leave an empty slot (and probably a comment explaining why it's skipped)